lib/sensors/main.cpp

```diff
@@ -0,0 +1,7 @@
+namespace hyped::sensors {
```

Comment:
just leave this file empty, we need a cpp file in the directory but don't need it to have anything. 

This file should be deleted in one of the temperature or accelerometer prs anyways as soon as we have an actual cpp file in here

Reply:
there shouldn't be anything called `main.cpp` or `main.hpp` imo

Reply:
Maybe an empty `sensors.cpp`?

Reply:
yeah will delete after either temp or acc get approved and merged into here

Reply:
@SnickeyX we don't need this pr to be open until the others are done. Replace main.cpp with an empty sensors.cpp, we merge this pr and then the other prs automatically shift bases to master.

Reply:
W